docs/source/api/tabular.rst

```diff
@@ -41,10 +41,9 @@ ___________
 
     ~forecasting.model.TabularForecaster
     ~forecasting.data.TabularForecastingData
+    ~forecasting.input.TabularForecastingDataFrameInput
```

Comment:
This should probably be moved down and have the `~` removed

Reply:
Done !